scripts/setup-config-test-dependencies.sh

```diff
@@ -2,34 +2,32 @@
 
 set -eu
 
-if [ "$TEST_SUBDIR" == "configuration-tests" ]; then
```

Comment:
The name of the script should be changed. I believe it only installs the compiler, so the name should reflect that.

Reply:
https://github.com/0xSpaceShard/starknet-hardhat-plugin/pull/399/commits/69563cf9a33d784fb285730482ae0a250ee3a7f4